optuna/pruners/_successive_halving.py

```diff
@@ -19,10 +19,10 @@ class SuccessiveHalvingPruner(BasePruner):
 
     Note that, this class does not take care of the parameter for the maximum
     resource, referred to as :math:`R` in the paper. The maximum resource allocated to a trial is
-    typically limited inside the objective function (e.g., ``step`` number in `simple.py
-    <https://github.com/optuna/optuna/tree/c5777b3e/examples/pruning/simple.py#L31>`_,
+    typically limited inside the objective function (e.g., ``step`` number in `simple_pruning.py
+    <https://github.com/optuna/optuna/tree/c5777b3e/examples/simple_pruning.py#L31>`_,
```

Comment:
This URL specifies the commit id, so please leave it as it is. Or, we can replace it with the master branch as follows:
```suggestion
    <https://github.com/optuna/optuna/blob/master/examples/simple_pruning.py#L31>`_,
```